spk/sonarr/Makefile

```
@@ -3,7 +3,7 @@ SPK_VERS = $(shell date +%Y%m%d)
 SPK_REV = 2
 SPK_ICON = src/sonarr.png
 
-DEPENDS = cross/sonarr cross/libstdc++ cross/libicu cross/sqlite
+DEPENDS = cross/sonarr
```

did you intentionally remove cross/sqlite ?
```suggestion
DEPENDS = cross/sonarr cross/sqlite
OPTIONAL_DEPENDS = cross/libstdc++
```